### ISSUE TITLE
Incorrect precedence between is and comparison operators

### ISSUE THREAD

**Issue by Sjord** [tree-sitter/tree-sitter-c-sharp#161]
```
        var someObj = new object();
        var allowedValuesList = someObj is null
            ? default
            : new object();
```

Actual: `someObj is (null ? ... : ...)`
Expected: `(someObj is null) ? ... : ...`

**Comment by damieng** [comment]
Likely same cause or fix as #150

**Comment by initram** [comment]
We should check if this is also fixed now.
